src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -197,8 +197,9 @@ const DiscoverySearch = ({
             returnKeyType="search"
             onSubmitEditing={handleOnSubmitEditing}
             placeholder={placeholderText}
-            placeholderTextColor={COLORS.lightgrey}
+            placeholderTextColor={COLORS.gray400}
             style={styles.input}
+            keyboardAppearance="dark"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The placeholder text color of the `TextInput` component has been changed from light grey to gray400. The keyboard appearance has also been set to "dark". Make sure these changes align with the overall design and user experience.
```diff
- placeholderTextColor={COLORS.lightgrey}
+ placeholderTextColor={COLORS.gray400}
+ keyboardAppearance="dark"
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->